### PR TITLE
Create 2019-01-30-custom-safetyGraphics-workflows.md

### PR DESCRIPTION
**Pull request by jwildfire**
Summary of v1.1 release

### CODE REVIEW

File: _posts/2019-01-30-custom-safetyGraphics-workflows.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+---
+layout: post
+title: Custom safetyGraphics Workflows
+author: Jeremy Wildfire
+---
+
+The `safetyGraphics` R package now supports for custom workflows allowing users to preload their own charts and data sets in the safetyGraphics Shiny Application. The best way to learn more is the new [Custom Workflows vignette](https://github.com/SafetyGraphics/safetyGraphics/wiki/Vignette:-Custom-Workflows) - which has been briefly summarized below.
```
Comment (bzkrouse) [inline _posts/2019-01-30-custom-safetyGraphics-workflows.md]: suggest linking to CRAN instead - https://cran.r-project.org/web/packages/safetyGraphics/vignettes/customWorkflows.html

File: _posts/2019-01-30-custom-safetyGraphics-workflows.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+```
+setwd('/path/to/the/file/above')
+safetyGraphicsApp()
+```
+
+Once the app opens, click the charts tab to view the new custom "hello_world" chart. 
+
+<img src="https://user-images.githubusercontent.com/3680095/71821298-c0080980-305f-11ea-979e-6574ac30f706.png" style='max-width:700px'>
+
+Again, see the [Custom Workflows vignette](https://github.com/SafetyGraphics/safetyGraphics/wiki/Vignette:-Custom-Workflows) for details and more complex examples.
```
Comment (bzkrouse) [inline _posts/2019-01-30-custom-safetyGraphics-workflows.md]: again link to cran?

**Review by bzkrouse — APPROVED** [review]

File: _posts/2019-01-30-custom-safetyGraphics-workflows.md (hunk 1 above)
Comment (pburnsdata) [inline _posts/2019-01-30-custom-safetyGraphics-workflows.md]: change "now supports for custom" to "now supports custom" or something like that

**Review by pburnsdata — APPROVED** [review]
WOOTS

File: _posts/2019-01-30-custom-safetyGraphics-workflows.md (hunk 1 above)
Comment (jwildfire) [inline _posts/2019-01-30-custom-safetyGraphics-workflows.md]:
```suggestion
The `safetyGraphics` R package now supports for custom workflows allowing users to preload their own charts and data sets in the safetyGraphics Shiny Application. The best way to learn more is the new [Custom Workflows vignette](https://cran.r-project.org/web/packages/safetyGraphics/vignettes/customWorkflows.html) - which has been briefly summarized below.
```

Comment (jwildfire) [inline _posts/2019-01-30-custom-safetyGraphics-workflows.md]:
```suggestion
The `safetyGraphics` R package now supports for custom workflows allowing users to preload their own charts and data sets in the safetyGraphics Shiny Application. The best way to learn more is the new [Custom Workflows vignette](https://cran.r-project.org/web/packages/safetyGraphics/vignettes/customWorkflows.html) - which has been briefly summarized below.
```

Comment (jwildfire) [inline _posts/2019-01-30-custom-safetyGraphics-workflows.md]:
```suggestion
The `safetyGraphics` R package now supports custom workflows allowing users to preload their own charts and data sets in the safetyGraphics Shiny Application. The best way to learn more is the new [Custom Workflows vignette](https://github.com/SafetyGraphics/safetyGraphics/wiki/Vignette:-Custom-Workflows) - which has been briefly summarized below.
```

**Pull request merged**